### ISSUE TITLE
create `useCreatePatient` react-query hook and remove redux create patient code

### ISSUE THREAD

**Issue by jackcmeyer** [HospitalRun/hospitalrun-frontend#2378]
## 🚀 Feature Proposal
This feature is to create a `useCreatePatient` react-query mutation which will create a patient. 

## Motivation
- removing redux

## Technical Notes
- this will follow a similar pattern to `useRequestImaging`
- make sure to remove all redux code related to creating a patient

**Comment by djiwandou** [comment]
is this issue somehow related to this? @jackcmeyer 
it happens in localhost and also in https://staging.hospitalrun.io/patients/new -> Create New Patient -> Save, only when opened using Firefox (normal in Chrome)
```
error: true
message: "unknown"
name: "indexed_db_went_bad"
reason: "A mutation operation was attempted on a database that did not allow mutations."
stack: ""
status: 500
```
<img width="1440" alt="Screen Shot 2020-09-10 at 11 16 32" src="https://user-images.githubusercontent.com/24618908/92681277-6e82ed80-f357-11ea-89c5-cb79ba962a33.png">

**Comment by jackcmeyer** [comment]
I don't think so! This code is to do a refactor of how the save of the patient works. 

I wasn't aware of any any bugs related to saving a patient. 

If this is a reproducible issue, could you open another bug please?

**Comment by djiwandou** [comment]
yes, it could be reproduced in my case it's only in Firefox
I will open new issue for this 🙏 
cheers

**Comment by fox1t** [comment]
@djiwandou thanks for pointing this out! Would you like to give it a shot?

**Comment by djiwandou** [comment]
wew I am still newbie, maybe not yet for this one hehe ✌️ 
@fox1t

**Comment by fox1t** [comment]
@djiwandou :)

**Comment by ArtemGoldsmith** [comment]
Hey! Can I take this issue?

**Comment by jackcmeyer** [comment]
Sounds good @ArtemGoldsmith!

**Comment by giulianovarriale** [comment]
I see this story was picked up long time ago. Can I take this one?
